libraries/AP_InertialSensor/AP_InertialSensor.h

```diff
@@ -31,6 +31,11 @@
 
 #define DEFAULT_IMU_LOG_BAT_MASK 0
 
+#ifndef HAL_INS_TEMPERATURE_CAL_ENABLE
+#define HAL_INS_TEMPERATURE_CAL_ENABLE !HAL_MINIMIZE_FEATURES && BOARD_FLASH_SIZE > 1024
```

Comment:
How about we allow this on 1MB boards for INS_MAX_INSTANCES == 1?

Reply:
quite possibly, yes, or at least on some boards